File: Upcoder/Other/Tìm số nguyên dương lớn nhất/LTCB-90.cs
using System;
namespace Timso
{
    public class Program
    {
        static void Main()
        {
            long n = long.Parse(Console.ReadLine());
            long tong = 0;
            long kq = 0;
    
            for (long i = 1; i < n; i++)
            {
                tong += i;
                if (tong > n)
                    break;
                kq = i;
            }
    
            Console.Write(kq);
        }
    }
}
    

File: Upcoder/Other/Tìm số nguyên dương lớn nhất/LTCB-90.cpp
#include <iostream>
using namespace std;
int main()
{
    long long n;
    cin>>n;
    long long tong = 0;
    long long kq;
    for (long long i = 1; i < n ; i++){
        tong+= i;
        if (tong > n)
            break;
        kq=i;
    }
    cout << kq;
}